### PR TITLE
add letterfx@1.0.2 w/ git auto-update

### LINKED ISSUE #9334: [Request] CanvasXpress
**Library name: canvasXpress** 
**Git repository url: https://github.com/neuhausi/canvasXpress**
**npm package url(optional):** 
**License(s): GLP3**
**Official homepage: http://canvasxpress.org/html/index.html**
**Wanna say something? Leave message here:**
# 

Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster than just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by kennynaoh**
Pull request for issue: #9224
Related issue(s): # #
@x09326 Please help me review this PR, thanks!
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/TuxSudo/letterfx
 * Official website (optional, not the repository): https://tuxsudo.com/code/project/letterfx
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 109
   - Count of watchers: 19
   - Count of forks: 38
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #9334, cc @tuxsudo

### CODE REVIEW

File: ajax/libs/letterfx/package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "letterfx",
+  "filename": "jquery-letterfx.min.css",
```
Comment (x09326) [inline ajax/libs/letterfx/package.json]: @kennynaoh 
Is this filename correct?
I think it is `jquery-letterfx.min.js`.

**Comment by kennynaoh** [comment]
@x09326 I have fixed it. Please help me review it again, thanks

**Comment by x09326** [comment]
LGTM
ping @PeterDaveHello

**Comment by PeterDaveHello** [comment]
What's the reason why to close a closed issue #9334 in the commit message? Looks like a typo, @x09326 I think you should take a look at it.
image

**Comment by kennynaoh** [comment]
@x09326 I have fixed the commit message. Please help me review it again, thanks!

**Comment by x09326** [comment]
@kennynaoh sorry for the late response.
Please rebase with the latest master branch, thanks.

**Comment by kennynaoh** [comment]
@x09326 I have rebased to the latest master branch. Please help me review it again. Thanks!

**Comment by x09326** [comment]
LGTM
ping @PeterDaveHello

**Pull request merged**